src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int GROW = 2;
+    private T[] elements;
+    private int maxSize = 10;
```

Comment:
[Make it constant](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/array-list.html#any-magic-numbers-should-be-constants)